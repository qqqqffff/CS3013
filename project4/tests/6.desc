eval SJF policy with 20 jobs with random length and tie breaker
