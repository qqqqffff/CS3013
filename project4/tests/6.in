11
6
1
14
5
6
8
13
20
10
20
20
5
2
4
17
12
19
20
14