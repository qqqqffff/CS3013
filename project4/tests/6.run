./scheduler SJF tests/6.in 0
