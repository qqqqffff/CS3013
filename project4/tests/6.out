Execution trace with SJF:
Job 2 ran for: 1
Job 13 ran for: 2
Job 14 ran for: 4
Job 4 ran for: 5
Job 12 ran for: 5
Job 1 ran for: 6
Job 5 ran for: 6
Job 6 ran for: 8
Job 9 ran for: 10
Job 0 ran for: 11
Job 16 ran for: 12
Job 7 ran for: 13
Job 3 ran for: 14
Job 19 ran for: 14
Job 15 ran for: 17
Job 17 ran for: 19
Job 8 ran for: 20
Job 10 ran for: 20
Job 11 ran for: 20
Job 18 ran for: 20
End of execution with SJF.
